WcaOnRails/config/locales/time_will_tell.ru.yml

```diff
@@ -0,0 +1,8 @@
+﻿en:
```

Comment:
This is the relevant part of the travis failure: https://travis-ci.org/thewca/worldcubeassociation.org/builds/196437173#L2000-L2006